test/albums.js

```diff
@@ -54,3 +54,58 @@ describe('/albums GET', () => {
       });
   });
 });
+
+describe('/albums/:id POST', () => {
+  it('should create a purchase', done => {
+    chai
+      .request(server)
+      .post('/users/sessions')
+      .send({
+        email: 'federico.casares@wolox.com.ar',
+        password: '12345678'
+      })
+      .then(response => {
+        chai
+          .request(server)
+          .post('/albums/1')
+          .set('authorization', `Bearer ${response.body.token}`)
+          .end((err, res) => {
```

Comment:
let's use promises instead of callbacks.

Also we are only checking the endpoint response and not if the logic is fine.

Reply:
Will refactor that in another pr.

Reply:
Done in #15